client/core/src/com/cows/game/controllers/BulletController.kt

```diff
@@ -0,0 +1,43 @@
+package com.cows.game.controllers
+
+import com.badlogic.gdx.math.Vector2
+import com.cows.game.models.BulletModel
+import com.cows.game.views.BulletView
+import kotlin.math.abs
+
+class BulletController(private val model: BulletModel) : Updatable() {
+
+    private var currentTarget: UnitController? = null
+
+
+    override fun update(deltaTime: Float) {
+        if (model.hasHit) return
+        if (currentTarget == null) return
+        val targetPos = currentTarget!!.model.position
+        val oldX = model.position.x
+        val oldY = model.position.y
+
+        model.position.x += ((targetPos.x + 32f - model.position.x)) / model.movementSpeed
```

Comment:
Is this the bullet speed?

Reply:
It moves the bullet towards the target, and how much it moves depends on the bullet speed.

Reply:
Ah okay, but 32f is the speed?

Reply:
no, 32f is to move it to the center of of the unit. Should probably have a getSize() or getCenterPosition() method in the Unit MVC if units will have different sizes. If this 32f is not added then the bullet would target the lower left corner of a unit

Reply:
Okay doke

Reply:
![image](https://user-images.githubusercontent.com/43535195/164406231-42cf3706-55d6-49a3-a352-29a6b6e2503d.png)

Basically like this :)

Reply:
Nice!